### DISCUSSION TITLE
How do I change the title from 'Siglas' to 'Siglas e Acrónimos'?

### DISCUSSION THREAD

**Discussion by csioliveira** [joaomlourenco/novathesis#481] [Q&A] [answered]
Hello, I am struggling to change the name of the glossary from 'Siglas' to 'Siglas e Acrónimos.' Please help me!! I also have symbols besides Siglas.

**Comment by joaomlourenco** [comment]
Always read the documentation! 😊
https://github.com/joaomlourenco/novathesis/wiki/FAQ#2-how-to-rename-the-list-ofsomething-to-list-of-whatever

*I hope this solution is still working… if it is, please mark select this as the answer! If it is not, please let me know and I'll double check!*

**Reply by csioliveira** [reply]
Hello, :) thank you for your response, but it didn't work. Could you please double-check?
Thank you once again

**Reply by joaomlourenco** [reply, marked as answer]
Solved. The correction will be available in the next release.  Meanwhile, just download (and replace) these two files:

- [novathesis.cls](https://github.com/joaomlourenco/novathesis/blob/develop/novathesis.cls)
- [NOVAthesisFiles/fix-babel.tex](https://github.com/joaomlourenco/novathesis/blob/develop/NOVAthesisFiles/fix-babel.tex)

and in file `Config/5_packages.tex` add `\ntlangsetup*{pt/acronym=Siglas e Acrónimos}`

Let me know it now it works for you!

**Reply by csioliveira** [reply]
yes it worked! thank you so much :)
